gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -219,6 +219,13 @@ typedef struct _GstTensorFilterFramework
        * @param[in] hw backend accelerator hardware
        * @return 0 if supported. -errno if not supported.
        */
+
+  int (*allocateInInvoke) (void **private_data);
```

Comment:
Either this will have to be made mandatory, or make an assumption that undefined function implies allocate in invoke is not supported for this framework.

Reply:
IMO, optional. If not defined, allocate-invoke is false and tensor-filter main will allocate the output buffer.

Reply:
Yup. That's doable. However, we now need to think about backward compatibility. We have private plugins that uses Tizen 5.5 and should be compatible with Tizen 6.0.

Choice1: Create data structure to support old plugins (w/o recompiling?)
Choice2: Some macro for major changes in filter subplugin data structure?
(e.g., #define NNS_FILTER_API (1) ?)

Reply:
I will not remove `allocate_in_invoke` till #2045 is resolved.

Reply:
@jaeyun-jung as per [discussion](https://github.com/nnsuite/nnstreamer/pull/2013#issuecomment-576140354), not remove `allocate_in_invoke` flag for now.

Reply:
Got it. Now we need to consider the changes of destroyNotify callback for the backward compatibility.

Reply:
Or... possibly...  assuming that we can ignore backward binary compatibility for now, we can remove the flag and define NULL of allocateininvoke callback means it's FALSE.